src/components/Card/Card.stories.mdx

```diff
@@ -0,0 +1,529 @@
+</Preview>
+
+## Cards in a Grid
+
+<Preview>
+  <Story name="Cards in a Grid">
+    <div
+      style={{
+        display: "grid",
+        gap: "2rem",
+        "grid-template-columns": "repeat(3, 1fr)",
+      }}
+    >
+      <Card>
+        <CardImage
+          src="https://placeimg.com/400/200/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/410/210/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/420/220/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/430/230/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/440/240/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+      <Card>
+        <CardImage
+          src="https://placeimg.com/450/250/animals"
+          alt="Alt text"
+          imageAspectRatio={CardImageRatios.TwoByOne}
+        />
+        <CardHeading level={3} id="heading1">
+          Card Heading
+        </CardHeading>
+        <CardContent>
+          Vestibulum id ligula porta felis euismod semper. Nulla vitae elit
+          libero, a pharetra augue.
+        </CardContent>
+      </Card>
+    </div>
+  </Story>
+</Preview>
+
+## Cards in a List
```

Comment:
This is technically not a list and should use `ul`, `li` elements.

Reply:
I changed this title to `Cards in a Stack`.